src/main/java/uk/gov/cabinetoffice/csl/controller/signup/SignupController.java

```diff
@@ -81,6 +82,8 @@ public class SignupController {
 
     private final ICivilServantRegistryClient civilServantRegistryClient;
```

Comment:
civilServantRegistryClient reference may not be required any more because CsrsService is used now.